mongo/integration/unified/admin_helpers.go

```diff
@@ -17,8 +17,12 @@ import (
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
-const (
-	errorInterrupted int32 = 11601
+var (
+	ignoredKillAllSessionsErrors = []int{
+		11601, // Interrupted, for SERVER-38335 on server versions below 4.2
+		13,    // Unauthorized, for SERVER-54216 on atlas
```

Comment:
So, by ignoring this `Unauthorized` error, we don't have to verify we aren't connected to an Atlas cluster when calling `killAllSessions` (i.e. look for `mongodb.net` in the hostname)?

Reply:
Yup, now it'll just ignore that error so the test can continue